scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+set -ex
+
+export GOROOT=/usr/local/go
+export PATH=$GOROOT/bin:$PATH
+
+export GOPATH=/eirini-release
+export PATH=$PWD/bin:$PATH
+
+rm -rf /tmp/*
+go get github.com/onsi/ginkgo/ginkgo
+
+pushd src/code.cloudfoundry.org/eirini/integration/recipe
+  ginkgo .
+popd
```

Comment:
Since the PR in eirini removes [integration/recipe](https://github.com/cloudfoundry-incubator/eirini/tree/f8d2eaef27caf7b384792e56d5315a72557925d6/integration/recipe), we think that this script should be moved to eirini-staging instead.